shared/profile/index.native.js

```diff
@@ -418,31 +418,28 @@ class Profile extends Component<Props, State> {
     if (section.title === 'profile') {
       const trackerStateColors = stateColors(this.props.currentlyFollowing, this.props.trackerState)
       return (
-        <Kb.Box
+        <Kb.HeaderHocHeader
+          borderless={true}
+          onLeftAction={this.props.onBack}
           style={{
```

Comment:
Shouldn't it be:
```suggestion
          headerStyle={{
```

I haven't tested that, but that might fix @buoyad issue

Reply:
Yup! Thanks for catching that. 🙇